solutions/usaco-265.mdx

```diff
@@ -0,0 +1,108 @@
+---
+id: usaco-265
+source: Old Gold
+title: The Cow Run
+author: Ryan Chou
+---
+ 
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_cowrun_mar13.html)
+
+## Explanation
+
+When we go from one point to another, it's optimal to pick up every cow along the way, so the list of cows that we visit will always form a subarray in the sorted list of a cows.
+
+So, we'll sort the cows by their position.
+
+However, the optimal solution might involve going from one side to another as well. To account for this, let's use [range DP](https://usaco.guide/gold/dp-ranges)
+
+#### States
+
+We have to keep track of the cows that we've used. Since this'll always be a range, we'll represent it as $i$ and $j$ (the endpoints of the interval).
```

Comment:
```suggestion
We have to keep track of the cows that we've already picked up. Since this'll always be a range, we'll represent it as $i$ and $j$ (the endpoints of the interval).
```